ci/common/test.sh

```diff
@@ -86,19 +86,16 @@ check_sanitizer() {
   fi
 }
 
-run_unittests() {(
-  enter_suite unittests
+run_unittests() {
```

Comment:
Why were all the subshells removed?

Reply:
Excellent question: I don't remember. I was convinced I needed to remove them but I tested it again with the subshells and it works. I'll add a fixup commit.